docs/bash-complete.rst

```diff
@@ -0,0 +1,11 @@
+.. _bash-complete:
+
+Bash Complete
```

Comment:
Could you rename this section "Bash Completion" instead please?  That's the more common nomenclature in my experience :)